README.md

```diff
@@ -14,6 +14,15 @@ tweetnacl.h -> 02aee579efbe4f12ad8ffb23c6772841
 ## Fixes
 Change integer size in commit [b98e0ca](https://github.com/RIOT-OS/tweetnacl/commit/b98e0ca8beb9b802b5c0492d88846bd9de9946eb)
 
+Fix a warning on line 58 about comparison between two different integer
+sizes.
```

Comment:
> If your argument is based on the warning output given by the compiler (which is about signedness), then I can put the complete output from the compiler.

Yes, I'm talking about the compiler warning, since it seems to me that is what you are referring to in this description. But maybe I'm mistaking. You don't have to put the explicit warning here.

> So, if while compiling for a 8bit platform on one side you have int and in the inner function you have a u32 variable there would not be an error, but we would be using a variable bigger than necessary (yes, its only 2 bytes, but still...).

Inner function? All I see is a macro that resolves to a `for`-loop like `for (usign = 0; usign < sign; usign++)`, but okay... let's put this discussion to rest, your argument seems sensible.